documentation/asciidoc/computers/remote-access/vnc.adoc

```diff
@@ -58,42 +56,35 @@ sudo raspi-config
---
-5. Click the "Connect" button to initiate a connection with the server.
-   * If TigerVNC warns you that the "Hostname does not match the server certificate", click the "Yes" button to continue.
+1. Download https://www.realvnc.com/download/viewer/[VNC Viewer]. You are entitled to use RealVNC's cloud service for free, provided that remote access is for educational or non-commercial purposes only.
+2. On your client device, launch VNC Viewer.
+3. Read and accept the terms and conditions.
+4. Click the *Use RealVNC Viewer without signing in* link to connect only to devices on your local network without RealVNC's cloud service.
+5. In the *Enter a VNC Server address or search* field at the top of the window, enter the hostname or IP address of your Raspberry Pi.
 +
 --
-image::images/vnc-tigervnc-cert-warning.png[TigerVNC warning about mismatched certificates,400]
+image::images/vnc-client-enter-ip.png[Entering the Raspberry Pi's local IP address into the VNC client.]
 --
-* If TigerVNC warns you that the "certificate has been signed by an unknown authority", click the "Yes" button to grant an exception for your Raspberry Pi.
+6. Press the *Enter* key to begin connecting.
+7. You may see an *Identity Check* window that warns that RealVNC Viewer has no record of this server. If this is the first connection to this device over VNC, this is expected. Press the *Continue* button to continue connecting.
 +
 --
-image::images/vnc-tigervnc-cert-signer-warning.png[TigerVNC warning about certificates signed by an unknown authority,400]
+image::images/vnc-client-no-record-continue.png[A warning that the VNC client has no record of the server.,400]
 --
-6. When prompted for a username and password, enter your credentials.
+8. In the *Authentication* window, enter the username and password for the user account on your Raspberry Pi.
 +
 --
-image::images/vnc-tigervnc-username-password.png[Entering a username and password to authenticate via TigerVNC,400]
+image::images/vnc-client-enter-credentials.png[Entering Raspberry Pi user account credentials into the VNC client.]
 --
-7. Click the "OK" button to authenticate with the VNC server. If your credentials are correct, TigerVNC should open a window containing the desktop corresponding to your account on the Raspberry Pi. You should be able to move your mouse and keyboard to input text and interact with the desktop.
+9. Press the *OK* button to connect to your Raspberry Pi.
+10. VNC Viewer will open a window containing the desktop for your account on the Raspberry Pi. You should be able to interact with the desktop using your mouse and keyboard automatically; you may have to enable accessibility access to use specialised keys such as media keys.
 +
 --
-image::images/vnc-tigervnc-desktop.png[The desktop of a Raspberry Pi after successfully authenticating with TigerVNC]
+image::images/vnc-client-desktop.png[The desktop of a Raspberry Pi after successfully authenticating.]
 --
```

Comment:
```suggestion
--

NOTE: If you authenticate successfully but the desktop display window closes immediately, check the Expert settings on VNC Viewer. If *ColorLevel* is set to *pal8*, change it to any other value, and VNC Viewer should display the desktop properly next time you connect.

```

Here's a note we can add to help with this pal8 bug. If some devices default to this color setting, I figure it's worth a note to help folks out, especially since this error only occurs *after* a successful authentication.